aspnetcore/mvc/razor-pages/razor-pages-convention-features/sample/Pages/_Layout.cshtml

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>@ViewData["Title"] - ModelProvidersSample</title>
+
+    <environment include="Development">
+        <link rel="stylesheet" href="~/lib/bootstrap/dist/css/bootstrap.css" />
+        <link rel="stylesheet" href="~/css/site.css" />
+    </environment>
+    <!--
+        Don't use the ajax.aspnetcdn.com CDN in production.
+    -->
```

Comment:
Convert from an HTML comment to a Razor comment to prevent this from displaying in the page source.